chrome_extension/record.py

```diff
@@ -1,14 +1,46 @@
 import asyncio
 import websockets
+import sqlite3
+import json
 
-async def print_message(message):
+# Function to initialize the database
+def init_db():
+    conn = sqlite3.connect('chrome.db')
+    c = conn.cursor()
+    c.execute('''
+        CREATE TABLE IF NOT EXISTS browser_event (
+            id INTEGER PRIMARY KEY AUTOINCREMENT,
+            timestamp DATETIME DEFAULT CURRENT_TIMESTAMP,
+            message TEXT
+        )
+    ''')
+    conn.commit()
+    conn.close()
```

Comment:
@KrishPatel13 can you please clarify why this is necessary/preferable compared to the standard procedure of generating alembic migrations?

Reply:
This is jsut a prototype I made for testing writing stuff into db. I am not going to use this function when integrating this into ouver main openadapt dir files. I hope this is answers the question. I will follow the standary way of creating and adding browser events as window events are done.